pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -235,7 +235,14 @@ public static void start(String[] args, SendGridSupplier sendGridSupplier) {
         setupFileScanResultReceiver(cfg, pubSubProject);
 
         final PubSubConnectionManager pubsubConnectionManager = new PubSubConnectionManager(usePubSubEmulator);
-
+        if (usePubSubEmulator) {
+            ProjectTopicName projectTopicName = ProjectTopicName.of(pubSubProject,
```

Comment:
Just curious what this fixes... was the PubSubConnectionManager instance not initialized properly in the emulator case?
How does this info find its way into PubSubConnectionManager in the non-emulator case?

Reply:
The connection manager is instantiated just fine.  It's the Service at line 246 that wasn't able to initialize properly because the task/subscription wasn't present.

These are manually created topics/subscriptions in each of the environments PEPPER runs in and is not in the codebase when running outside of the emulator.